src/components/activity/ActivityNotification.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, useId } from 'react';
+import { useNotifications } from 'hooks/useNotifications';
+import { NotificationActivity } from 'libs/notifications/types';
+import { activityActionName, activityDescription } from './utils';
+import { ActivityIcon, ActivityId } from './ActivityTable';
+import { ReactComponent as IconClose } from 'assets/icons/X.svg';
+import { useInterval } from 'hooks/useInterval';
+import { FOUR_SECONDS_IN_MS } from 'utils/time';
+import { Link } from '@tanstack/react-router';
+
+interface Props {
+  notification: NotificationActivity;
+  isAlert?: boolean;
+}
+
+export const ActivityNotification: FC<Props> = ({ notification, isAlert }) => {
```

Comment:
nit: This component looks like a duplicate of the existing component in terms of logic - maybe we can extract this into a base component and then just pass the markup down. If the base component returns a the children as a function, then you can access the `handleCloseClick` in the parents. Not sure if I explained this correct - shoot me a msg if not clear what I am trying to say :D

Reply:
The feature is deprecated in v5.